src/crimson/os/seastore/cached_extent.h

```diff
@@ -415,6 +415,13 @@ class CachedExtent
     return is_mutable() || state == extent_state_t::EXIST_CLEAN;
   }
 
+  /// Returns true if extent's parent is stable
```

Comment:
/// Returns true if extent is stable and shared among transactions

Reply:
done